Model/ShimModel.php

```diff
@@ -29,6 +29,16 @@ public function __construct($id = false, $table = null, $ds = null) {
 			$this->prefixOrderProperty();
 		}
 
+		if ($warn = Configure::read('Shim.warnAboutRelationProperty')) {
+			if (!empty($this->getAssociated())) {
```

Comment:
Just !$... suffices.

Reply:
You mean `if ($this->getAssociated()) {` ?
Having the ! returns false when there are associations.